packages/components/pie-button/src/button.scss

```diff
@@ -72,7 +72,7 @@
     }
 
     position: relative;
-    display: flex;
+    display: inline-flex;
```

Comment:
Even though the button was correctly sized, the web component host was a block element so you could click next to the button and it would still register the click.

e.g., here: https://aperture-nuxt.pie.design/components/button